lib/dogma/rules/hard_tabs.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Dogma.Rules.HardTabs do
+  @moduledoc """
+  Requires that all indentation is done using spaces rather than hard tabs.
+
+  So the following would be invalid:
+
+  def something do
+  \t:body
```

Comment:
Yeah I'm still not sure if I actually want a tab in the documentation. The fact that all blank space looks the same makes it kind of useless as documentation.  How would you feel if it read like:

```
 So the following would be invalid:

  def something do
  {tab} :body
```